flow-libs/terser.js.flow

```diff
@@ -28,7 +28,7 @@ declare module 'terser' {
         ecma?: ECMAVersion,
```

Comment:
Maybe this options type should just be `$Shape<{|...|}>` as everything is optional and it may be acceptable to use an empty object (which I believe isn't possible with an exact object type with all optional properties).

Reply:
Yeah we could use that, but flow passed so gonna leave it for now